src/Uno.UI/UI/Xaml/Media/Animation/Animators/RenderingLoopAnimator.wasm.cs

```diff
@@ -15,17 +16,57 @@ protected RenderingLoopAnimator(T from, T to)
 
 		public JSObjectHandle Handle { get; }
 
+		protected override void EnableFrameReporting()
+		{
+			if (Handle.IsAlive)
+			{
+				WebAssemblyRuntime.InvokeJSWithInterop($"{this}.EnableFrameReporting();");
+			}
```

Comment:
I think we should still add debug logs (they'll be removed by the linker soon anyways)